src/org/salesforce/apexdoc/ApexDoc.java

```diff
@@ -397,6 +397,7 @@ private static void fillPropertyModel(PropertyModel propertyModel, String name,
     private static void fillMethodModel(MethodModel mModel, String name, ArrayList<String> lstComments, int iLine) {
         mModel.setNameLine(name, iLine);
         boolean inDescription = false;
+        boolean inExample = false;
```

Comment:
below inside the loop, for every line that sets inDescription = false when a new @token is found, should also set inExample = false?